core/sbt-cloudflow/src/main/scala/cloudflow/sbt/CloudflowBasePlugin.scala

```diff
@@ -58,6 +59,58 @@ abstract class CloudflowBasePlugin extends AutoPlugin {
   override def requires = CommonSettingsAndTasksPlugin && StreamletScannerPlugin &&
     JavaAppPackaging && sbtdocker.DockerPlugin
 
+  override def projectSettings = Seq(
+    libraryDependencies ++= Vector(
+      "com.lightbend.cloudflow" % "cloudflow-runner"         % BuildInfo.version
```

Comment:
This has to be `%` and not `%%` since we need the runner jar without any version information for Flink. Hence we build the runner jar from `core` without the version information.

Reply:
Might be worth a brief inline comment to explain this to future readers of the code.

[very minor] also, the big gap between the artifact id and the version is weird since there is nothing to align with.

Reply:
I will add the comment. Regarding the big gap I think it's from `scalafmt` ..